### PR TITLE
Turning in Assignment

### PR DESCRIPTION
**Pull request by steftheo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="CSS/style.css"></link>
+
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <div class="row">
+
+          <div class="column">
```
Comment (rtablada) [inline index.html:13]: Looks like there may be some unclosed divs here.

Delete the code for the second column and by placing your cursor on the opening tag of this first `column` element, your editor will underline the closing tag.

Then use the knowledge of where the opening and closing tags are to duplicate the full element and then work from there.

**Comment by steftheo** [comment]
Fixed

**Comment by rtablada** [comment]
@steftheo check to make sure your latest changes have been pushed here.

**Comment by steftheo** [comment]
pushed

**Pull request merged**